js/mail.js

```diff
@@ -631,6 +631,14 @@ var Mail = {
 					data.replyCc = draft.cc;
 					data.replyBcc = draft.bcc;
 					data.replyBody = draft.body;
+				} else {
+					// Add body content to inline reply (text mails)
+					if(!data.hasHtmlBody) {
```

Comment:
Yep. I also thought before that `if(` was better style but the JS guidelines from both Mozilla and Google say `if (`. :)